### PR TITLE
Add axes to user inventory

### PR DESCRIPTION
**Pull request by JiangfengLi**
Add axes to both tutorial and main mission from "addItemToInventory" function of "Mission" class and "addItemsToInventory" of "SARMission" class.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.10)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+
```
Comment (adarshp) [inline CMakeLists.txt:6]: Can you remove this extra line?

File: external/malmo/Minecraft/launchClient.sh [hunk 2]
```diff
@@ -68,8 +68,8 @@ fi
 
 # Finally we can launch the Mod, which will load the config file
 while
-    ./gradlew setupDecompWorkspace
-    ./gradlew build
-    ./gradlew runClient
+    ./gradlew setupDecompWorkspace --stacktrace
```
Comment (adarshp) [inline external/malmo/Minecraft/launchClient.sh:71]: Can you remove the `--stacktrace` options in this file? They are for debugging purposes and we don't need the end-user to see the stack traces.

File: external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Mission.java [hunk 3]
```diff
@@ -585,4 +587,17 @@ public void setTimeLimitInSeconds(long timeLimitInSeconds) {
   public void addToDeletion(Entity entity, long worldTime) {
     this.entitiesToRemove.put(entity, worldTime);
   }
+
+    /**
+     * Add items to the player's inventory to help them accomplish the mission
```
Comment (adarshp) [inline external/malmo/Minecraft/src/main/java/edu/arizona/tomcat/Mission/Mission.java:592]: Can change 'items' to 'item'? (Since it's a single item that is being added.)

File: facelandmarkdetection/drawLandmarks.hpp [hunk 4]
```diff
@@ -0,0 +1,72 @@
+//
```
Comment (adarshp) [inline facelandmarkdetection/drawLandmarks.hpp:1]: This file should not be part of the PR - can you remove it?

File: facelandmarkdetection/faceLandmarkDetection.cpp [hunk 5]
```diff
@@ -0,0 +1,72 @@
+//
```
Comment (adarshp) [inline facelandmarkdetection/faceLandmarkDetection.cpp:1]: Please remove this file as well.

File: CMakeLists.txt [hunk 6]
```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
 
-set(CMAKE_CXX_STANDARD 17)
```
Comment (adarshp) [inline CMakeLists.txt:3]: @JiangfengLi Why have these lines been removed?

File: external/malmo/Minecraft/run/options.txt [hunk 7]
```diff
@@ -1,102 +0,0 @@
-version:922
```
Comment (adarshp) [inline external/malmo/Minecraft/run/options.txt:1]: @JiangfengLi Please checkout the version of this file from the master branch with:

    git checkout master -- external/malmo/Minecraft/run/options.txt

File: CMakeLists.txt (hunk 6 above)
Comment (JiangfengLi) [reply CMakeLists.txt:3]: I thought you told me to remove that

Comment (adarshp) [reply CMakeLists.txt:3]: @JiangfengLi I meant the blank lines..

Comment (adarshp) [reply CMakeLists.txt:3]: Go ahead and do 

```
git checkout master -- CMakeLists.txt
```

Comment (adarshp) [reply CMakeLists.txt:3]: (Followed by `git push`)
